src/main/java/com/jalasoft/webservice/controller/DownloadController.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.jalasoft.webservice.controller;
+
+import com.jalasoft.webservice.utils.CheckSum;
```

Comment:
remove not used imports:
import com.jalasoft.webservice.utils.CheckSum;
import com.jalasoft.webservice.utils.FileManager;
import com.jalasoft.webservice.utils.PropertiesReader;
import javax.activation.MimetypesFileTypeMap;
import static com.jalasoft.webservice.utils.Constants.APPLICATION_PROPERTIES;